packages/flutter/lib/src/material/slider.dart

```diff
@@ -431,6 +433,12 @@ class Slider extends StatefulWidget {
   /// {@macro flutter.widgets.Focus.autofocus}
   final bool autofocus;
 
+  /// {@macro flutter.cupertino.slider.thumbColor}
+  ///
+  /// Used if this slider is created with [Slider.adaptive] and
+  /// running in a cupertino style platform.
+  final Color thumbColor;
```

Comment:
should we call this cupertinoThumbColor to be clear about its purpose?

cc @HansMuller

Reply:
I actually thought about this just did not used this name because I was thinking to propose to provide a way to change the thumb color for Android as well (if this a valid  feature).

Reply:
`thumbColor` is on the `SliderTheme`, but the `Slider` constructor has a more general `activeColor` which the thumb color inherits from. If this is meant to affect both the Material and Cupertino slider thumb color, then it should probably be aligned with the `thumbColor` param in the `SliderTheme`. But if it is just for the `CupertinoSlider`, then I agree that calling it `cupertinoSliderThumb` would be more clear, and it can also only be exposed in the `Slider.adaptive` constructor.

Reply:
> and it can also only be exposed in the `Slider.adaptive` constructor.

I can't find an simple way of doing this since both constructors are `const` and  they require that all property must be final. Isn't the usage description in the property enough?

Anyway, creating a instance with the primary constructor this property will never be used.

Reply:
> and it can also only be exposed in the `Slider.adaptive` constructor.

Iif you have any tip in how I can achieve this please tell me. I really want to know.

Reply:
Just updated the property name to `cupertinoThumbColor`